doc/pages/user/017-astarte_grafana_plugin.md

```diff
@@ -0,0 +1,49 @@
+# Astarte Datasource Plugin for Grafana
+
+Astarte Datasource Plugin conveys data from Astarte to
```

Comment:
`conveys` -> `exports`, maybe?

Reply:
exports is never used when talking about retrieving data from an API. In this context conveys feels more appropriate. Maybe **retrieve** is an other option, but i like the former more.